tests/Fixer/Whitespace/ArrayIndentationFixerTest.php

```diff
@@ -0,0 +1,563 @@
+<?php
+
+/*
+ * This file is part of PHP CS Fixer.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *     Dariusz Rumiński <dariusz.ruminski@gmail.com>
+ *
+ * This source file is subject to the MIT license that is bundled
+ * with this source code in the file LICENSE.
+ */
+
+namespace PhpCsFixer\Tests\Fixer\Whitespace;
+
+use PhpCsFixer\Tests\Test\AbstractFixerTestCase;
+use PhpCsFixer\WhitespacesFixerConfig;
+
+/**
+ * @internal
+ *
+ * @covers \PhpCsFixer\Fixer\Whitespace\ArrayIndentationFixer
+ */
+final class ArrayIndentationFixerTest extends AbstractFixerTestCase
+{
+    /**
+     * @param string      $expected
+     * @param null|string $input
+     *
+     * @dataProvider provideFixCases
+     */
+    public function testFix($expected, $input = null)
+    {
+        $this->doTest($expected, $input);
+    }
+
+    public function provideFixCases()
+    {
+        return $this->withLongArraySyntaxCases([
+            [
+                <<<'EXPECTED'
+<?php
+$foo = [
+    'foo',
+    'bar' => 'baz',
+];
+EXPECTED
+                ,
+                <<<'INPUT'
+<?php
+$foo = [
+  'foo',
+        'bar' => 'baz',
+ ];
+INPUT
+                ,
+            ],
```

Comment:
it shall pass, but please add
```php
            [
                <<<'EXPECTED'
<?php
$foo = [
    'foo', // comment
    'bar',
];
EXPECTED
                ,
                <<<'INPUT'
<?php
$foo = [
  'foo', // comment
'bar',
 ];
INPUT
                ,
            ],
```

just to be sure

Reply:
Added.